raw-spans-grouper/build.gradle.kts

```diff
@@ -33,11 +43,44 @@ tasks.test {
   useJUnitPlatform()
 }
 
+tasks.register<Copy>("gatherSourceAvdlFilesForCompatibilityCheck"){
+  from("src/main/avro/util/")
+  from({
+    zipTree(avdlDependency.singleFile)
+
+  }){
+    include("**/*.avdl")
+  }
+  into(file("$buildDir/tmp-source-avdl/raw-spans-grouper/"))
+}
+
+tasks.create<Copy>("gatherAgainstAvdlFilesForCompatibilityCheck") {
+  from(tasks.getByName<org.hypertrace.gradle.avro.CheckAvroCompatibility>("avroCompatibilityCheck").againstFiles) {
+    eachFile {
+      relativePath = RelativePath(true, *relativePath.segments.drop(1).toTypedArray())
+    }
+  }
+  from({
+    zipTree(avdlDependency.singleFile)
+  }) {
+    include("**/*.avdl")
+  }
+  into("$buildDir/tmp-against-avdl/raw-spans-grouper")
+}
+
+//tasks.named<org.hypertrace.gradle.avro.CheckAvroCompatibility>("avroCompatibilityCheck") {
```

Comment:
avroCompatibility check is failing when new avro files are introduced ...will look into it